src/features/walletconnect/components/WcProposalForm/index.tsx

```diff
@@ -162,7 +141,7 @@ const WcProposalForm = ({ proposal, setProposal }: ProposalFormProps): ReactElem
           {isLoading === WCLoadingState.REJECT ? <CircularProgress size={20} /> : 'Reject'}
         </Button>
 
-        <Button variant="contained" onClick={onApprove} className={css.button} disabled={disabled}>
+        <Button variant="contained" onClick={() => onApprove()} className={css.button} disabled={disabled}>
```

Comment:
```suggestion
        <Button variant="contained" onClick={onApprove} className={css.button} disabled={disabled}>
```

Reply:
`onApprove` takes an optional parameter and in this case we don't want to pass it but `onClick` passes an event object so we have to wrap the onApprove call.